Added resource method to update user details & updated default access (#132)
whirli/whirli-client: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

> - Adding resource for users to update user details.
> - Changed the default access on the users resource to be 'member' because it's now the majority of use cases.

Reviewer: Please add a heading for the type of changes these are, and please use imperative verbs for the changelog:

```suggestion
### Features
- Add resource for users to update user details.

### 🏡 Chore
- Change the default access on the users resource to be 'member'.
```

Reviewer: It's also nice to add a PR or issue number to the end too:

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]

> ### 🚀 Features
> - Add resource for users to update user details. [#132](https://github.com/whirli/whirli-client/pull/132)
> ### 🏡 Chore
> - Changed the default access on the users resource to be 'member'. [#132](https://github.com/whirli/whirli-client/pull/132)

Reviewer: The outer brackets were missed (just a convention we have)

```suggestion
### 🚀 Features
- Add resource for users to update user details. ([#132](https://github.com/whirli/whirli-client/pull/132))
### 🏡 Chore
- Changed the default access on the users resource to be 'member'. ([#132](https://github.com/whirli/whirli-client/pull/132))
```
